src/containers/Group/Producers/Item.js

```diff
@@ -16,6 +17,7 @@ class Item extends Component {
 
     return (
       <div>
+        <Link to={`/groups/1/producers/${producer.id}`}>hola</Link>
```

Comment:
Which should be the correct way? `./producers/${producer.id}` ?

Would be nice just `to={producer.id}` but it doesn't work